### DISCUSSION TITLE
Add padding to Child elements.

### DISCUSSION THREAD

**Discussion by rahadkc** [tailwindlabs/tailwindcss#2087] [Ideas]
Is there any class for add padding to child elements? I don't find any so far.
Sometimes instead of repeating list there're different types of child elements with similar padding and margin.

I know there are 'grid' 'gap' 'space-x-{$}' utility but I want a utility with padding like - 'padding-child-5',  'padding-child-right-5', 'pc-r-5', 'pc-rl-5', 'pc-5' etc. 
Thank you guys for your amazing library.

**Comment by adamwathan** [comment]
You might want to check out this plugin!

https://github.com/benface/tailwindcss-children

**Reply by EraAmate** [reply]
deprecated library 🔝
